php/src/Data/ConfigurationManager.php

```diff
@@ -188,6 +196,15 @@ public function SetBorgBackupHostLocation(string $location) : void {
         $this->WriteConfig($config);
     }
 
+    public function GetApachePort() : string {
+        $port = getenv('APACHE_PORT');
```

Comment:
Here is basically where you set everything up. Basically when starting the mastercontainer and set the environmental variable APACHE_PORT, apache and the domaincheck container will use that.

Reply:
TODO: write it to the config file if the file is present. And overwrite the value with the one given by the environmental varialbe. If not given use the one form config.json

Reply:
done